frontend/javascripts/oxalis/view/right-border-tabs/segments_tab/segment_list_item.tsx

```diff
@@ -455,7 +457,7 @@ function _SegmentListItem({
         padding: "2px 5px",
       }}
       className={classnames("segment-list-item", {
-        "is-selected-cell": segment.id === selectedSegmentId,
+        "is-selected-cell": selectedSegmentIds?.includes(segment.id), //TODO this wont work
```

Comment:
what exactly does this do? If I set it to false the segments are selected nonetheless?

Reply:
(will delete the TODO comment obviously)

Reply:
so this actually makes the coloring a little better, but the box thats created while hovering has a different size than the box thats created upon selection, so I will see whether I can maybe improve this.